bridge/client/app/app.module.ts

```diff
@@ -36,10 +36,12 @@ import { DtTagModule } from '@dynatrace/barista-components/tag';
 import { DtTopBarNavigationModule } from "@dynatrace/barista-components/top-bar-navigation";
 import { DtCopyToClipboardModule } from "@dynatrace/barista-components/copy-to-clipboard";
 import { DtToggleButtonGroupModule } from "@dynatrace/barista-components/toggle-button-group";
+import {DtQuickFilterModule} from "@dynatrace/barista-components/experimental/quick-filter";
```

Comment:
please align formatting of this import with the other imports (Note to myself: we need a linter :D )

Reply:
Yes, we should activate a linter tool. I create a follow-up issue for this, see #2915

Reply:
Prettier is maybe also interesting and could make it a bit easier to introduce style rules to the existing code base (sorry, I'm a Prettier-fanboy :smile: )